components/suggest/src/rs.rs

```diff
@@ -214,6 +202,15 @@ pub(crate) enum SuggestRecord {
     Geonames,
 }
 
+impl SuggestRecord {
+    fn to_json_map(&self) -> Map<String, Value> {
+        match serde_json::to_value(self) {
+            Ok(Value::Object(map)) => map,
+            _ => Map::new(),
```

Comment:
Maybe this branch could be `unreachable!()`.  I think we can safely say that `serde` will always be able to to deserialize to `Value::Object`.